doc/manual/metaprogramming.rst

```diff
@@ -9,106 +9,120 @@
 The strongest legacy of Lisp in the Julia language is its metaprogramming
 support. Like Lisp, Julia represents its own code as a data structure of
 the language itself.
+Because all data types and code in Julia are represented by Julia data structures,
+powerful reflection capabilities are available to explore the internals of a
```

Comment:
"reflection" might not be a well-known term among non-CS people. Link to e.g. wikipedia?